server/lib/rippled.js

```diff
@@ -105,3 +105,31 @@ module.exports.getHealth = async () => {
     }
   });
 };
+
+module.exports.getLedger = parameters => {
```

Comment:
This was here previously.

Reply:
Any idea why these files are appearing as additions if they were here before?

Reply:
It's here on `staging`, but is removed in `move-rippled`, so it's supposed to show up as an addition